augur/tasks/github/messages.py

```diff
@@ -14,7 +14,7 @@
 from sqlalchemy.sql import text
 
 platform_id = 1
-
+MESSAGE_BATCH_SIZE = 200
```

Comment:
@MoralCode / @shlokgilda : I think this amounts to all the requested changes I see.